### ISSUE TITLE
Missing Mod badge on posts and comments

### ISSUE THREAD

**Issue by BarterClub** [Memmy-App/memmy#610]
**Describe the bug
The mod tags are missing on posts and community you mod. 
Should be shown as a badge on post and comment. 

**To Reproduce**
1. Go to To any post you made and are a mod of the community. 

**Expected behavior**
Should show icon mod on post and comments. 

![image](https://github.com/Memmy-App/memmy/assets/10663563/77f1b6b5-f523-49f0-b825-0e89b94cd03f)

![image](https://github.com/Memmy-App/memmy/assets/10663563/066ccddc-089c-4c83-82f7-94fea9711a3f)

![image](https://github.com/Memmy-App/memmy/assets/10663563/41649651-0f27-4e4e-b175-8a207f25f3d7)

![image](https://github.com/Memmy-App/memmy/assets/10663563/c5e1b140-07fc-46d5-b31d-e91dd292c1f2)

**Comment by sgriff96** [comment]
I've noticed this with Admin tags as well.

**Comment by bohn002** [comment]
initial thought is there is not enough logic to handle multiple labels... so Dev->Admin->Op is the offramp pattern and the chip just seems to handle one option. Going to try and figure out a short option to handle multiple tags

**Comment by sgriff96** [comment]
Nah thats not it. We don't have who is a mod when we load the post. We'd have to query the community info to get that.

**Comment by bohn002** [comment]
yeah, just got there. that seems like a decent level of work for that call/logic. Maybe that might be worth doing as a separate issue.
